Dependencies/v8inspector/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+# ------------------------------- asio -------------------------------
+add_library(asio INTERFACE)
+target_include_directories(asio INTERFACE "Dependencies/asio/asio/include")
+set_property(TARGET asio PROPERTY FOLDER Dependencies)
+
+# ------------------------------ llhttp ------------------------------
+# Install node to use npm
+set(NODE_VERSION "v14.15.5")
+set_win32_arch()

```

Comment:
does this mean this v8inspector module won't work with Android?

Reply:
Good point-- this line doesn't exclude the ability to work on android, it just won't build on OSX or Linux, but this should be simple to fix in a follow-up PR, should just need to download the OSX/Linux version of node and run the correct command.

Reply:
I'm just going to limit ChromeDevTools to WIN32 for now so we don't run into any issues with Android until we have time to test its compatibility. In theory it should just work after some small fixes, but I wouldn't be confident without spending some time checking.